productive-seas-fish-and-shellfish-stocks-fished-sustainably/wrangling.py

```diff
@@ -0,0 +1,19 @@
+from email.policy import default
+import click
+import pandas as pd
+from pathlib import Path
+
+@click.command()
+@click.argument("input", type=click.Path(exists=True, path_type=Path))
+@click.option("--output", default=Path("./output.csv"), type=click.Path(path_type=Path))
+def wrangle(input: Path(), output: Path()) -> None:
+    
+    df = pd.read_csv(input, na_values="na")
+
+    df['Value'] = df['Value'].astype(float).round(2)
+
```

Comment:
You can delete the empty lines 12 and 14.
It is a short code; no need for the spaces.

Reply:
Done